mysql-test/main/mdev-30469.test

```diff
@@ -0,0 +1,131 @@
+#
```

Comment:
I think the name 'MDEV-30460.test' will not ring any bells for nearly anyone.  Please rename to something like 
'`delete_multi_order_by.test` ?

Reply:
Fixed.